2022-04-13 20:18:14,875 INFO    MainThread:41644 [wandb_setup.py:_flush():75] Loading settings from C:\Users\10194\.config\wandb\settings
2022-04-13 20:18:14,876 INFO    MainThread:41644 [wandb_setup.py:_flush():75] Loading settings from d:\deep_learing_work\Spynet\wandb\settings
2022-04-13 20:18:14,876 INFO    MainThread:41644 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-04-13 20:18:14,877 INFO    MainThread:41644 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-04-13 20:18:14,880 INFO    MainThread:41644 [wandb_init.py:_log_setup():405] Logging user logs to d:\deep_learing_work\Spynet\wandb\run-20220413_201814-21tqa4qr\logs\debug.log
2022-04-13 20:18:14,883 INFO    MainThread:41644 [wandb_init.py:_log_setup():406] Logging internal logs to d:\deep_learing_work\Spynet\wandb\run-20220413_201814-21tqa4qr\logs\debug-internal.log
2022-04-13 20:18:14,883 INFO    MainThread:41644 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000213826764C0>
2022-04-13 20:18:14,884 INFO    MainThread:41644 [wandb_init.py:init():439] calling init triggers
2022-04-13 20:18:14,885 INFO    MainThread:41644 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-04-13 20:18:14,886 INFO    MainThread:41644 [wandb_init.py:init():458] re-initializing run, found existing run on stack: m2p294lb
2022-04-13 20:18:14,893 INFO    MainThread:41644 [wandb_run.py:finish():1539] finishing run wtj/spynet/m2p294lb
2022-04-13 20:18:14,895 INFO    MainThread:41644 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-04-13 20:18:14,898 INFO    MainThread:41644 [wandb_run.py:_restore():1674] restore
2022-04-13 20:18:15,242 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1151
  total_bytes: 1151
}

2022-04-13 20:18:15,719 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1151
  total_bytes: 1151
}

2022-04-13 20:18:16,263 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1151
  total_bytes: 1151
}

2022-04-13 20:18:16,913 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1151
  total_bytes: 2875
}

2022-04-13 20:18:17,032 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1151
  total_bytes: 2875
}

2022-04-13 20:18:17,146 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1151
  total_bytes: 2875
}

2022-04-13 20:18:17,251 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1151
  total_bytes: 2875
}

2022-04-13 20:18:17,362 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1151
  total_bytes: 2875
}

2022-04-13 20:18:17,466 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1151
  total_bytes: 2875
}

2022-04-13 20:18:17,581 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1536
  total_bytes: 2875
}

2022-04-13 20:18:17,699 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1564
  total_bytes: 2875
}

2022-04-13 20:18:17,804 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1564
  total_bytes: 2875
}

2022-04-13 20:18:17,961 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2788
  total_bytes: 2875
}

2022-04-13 20:18:18,095 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2875
  total_bytes: 2875
}

2022-04-13 20:18:18,216 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2875
  total_bytes: 2875
}

2022-04-13 20:18:18,333 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2875
  total_bytes: 2875
}

2022-04-13 20:18:18,453 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2875
  total_bytes: 2875
}

2022-04-13 20:18:18,582 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2875
  total_bytes: 2875
}

2022-04-13 20:18:19,663 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2875
  total_bytes: 2875
}

2022-04-13 20:18:21,223 INFO    MainThread:41644 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2875
  total_bytes: 2875
}
local_info {
}

2022-04-13 20:18:22,904 INFO    MainThread:41644 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-04-13 20:18:22,904 INFO    MainThread:41644 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-04-13 20:18:22,918 INFO    MainThread:41644 [wandb_run.py:_footer_sync_info():2822] logging synced files
2022-04-13 20:18:22,948 INFO    MainThread:41644 [wandb_init.py:init():492] starting backend
2022-04-13 20:18:22,949 INFO    MainThread:41644 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-04-13 20:18:22,997 INFO    MainThread:41644 [backend.py:ensure_launched():219] starting backend process...
2022-04-13 20:18:23,677 INFO    MainThread:41644 [backend.py:ensure_launched():224] started backend process with pid: 30428
2022-04-13 20:18:23,682 INFO    MainThread:41644 [wandb_init.py:init():501] backend started and connected
2022-04-13 20:18:23,727 INFO    MainThread:41644 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-04-13 20:18:23,727 INFO    MainThread:41644 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-04-13 20:18:23,728 INFO    MainThread:41644 [wandb_init.py:init():565] updated telemetry
2022-04-13 20:18:24,119 INFO    MainThread:41644 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-04-13 20:18:32,500 INFO    MainThread:41644 [wandb_run.py:_on_init():1759] communicating current version
2022-04-13 20:18:33,258 INFO    MainThread:41644 [wandb_run.py:_on_init():1763] got version response upgrade_message: "wandb version 0.12.14 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-04-13 20:18:33,259 INFO    MainThread:41644 [wandb_init.py:init():625] starting run threads in backend
2022-04-13 20:18:34,148 INFO    MainThread:41644 [wandb_run.py:_console_start():1733] atexit reg
2022-04-13 20:18:34,151 INFO    MainThread:41644 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-04-13 20:18:34,151 INFO    MainThread:41644 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-04-13 20:18:34,161 INFO    MainThread:41644 [wandb_run.py:_redirect():1667] Redirects installed.
2022-04-13 20:18:34,161 INFO    MainThread:41644 [wandb_init.py:init():664] run started, returning control to user process
2022-04-13 20:18:34,518 INFO    MainThread:41644 [wandb_init.py:_pause_backend():318] pausing backend
2022-04-13 20:18:34,519 INFO    MainThread:41644 [jupyter.py:save_ipynb():374] not saving jupyter notebook
